### PR TITLE
Sashimi option for LFM added with --shirashi option

### PR DESCRIPTION
**Pull request by kkoetter**
Option for operating sashimi without laser or scanning added to support light field acquisition. Reworked config file creation and options to call in main.py as well as the state to work around enabling some functionalities for saving and triggering of stytra protocols.

For running the option:

sashimi --shirashi

### CODE REVIEW

File: sashimi/config.py [hunk 1]
```diff
@@ -27,6 +27,7 @@
     "email": {"user": "foo", "password": "foo"},
     "array_ram_MB": 450,
     "scopeless": False,
+    "shirashi": False,
```
Comment (vilim) [inline sashimi/config.py:30]: First why is this called "shirashi"? Google does not provide any results for this. Please replace with a more descriptive name like `camearonly` or `noscanning`.
Alternatively (and more cleanly) there can be `camera`, `laser` and `scanning` options, which would be all `True` for the lighsheet configuration. (there is a difference from scopeless, as this has mock classes to test all the functionality, and this is about disabling functionality altogether).

File: sashimi/scanning.py [hunk 2]
```diff
@@ -490,3 +490,38 @@ def run(self):
                 self.parameters = deepcopy(
                     scanloop.parameters
                 )  # set the parameters to the last ones received in the loop
+
+
+
+
+class MockScanner(Process):
```
Comment (vilim) [inline sashimi/scanning.py:497]: The mock classes are there for testing, a full solution for this requires a different version, perhaps `DisabledScanner`. Anyway I will work on it in the context of #79, you can leave it like this for the current PR.

File: sashimi/state.py [hunk 3]
```diff
@@ -430,7 +465,7 @@ def send_camera_settings(self):
         camera_params = convert_camera_params(self.camera_settings)
         camera_params.trigger_mode = (
             TriggerMode.FREE
-            if self.global_state == GlobalState.PREVIEW
+            if self.global_state == GlobalState.PREVIEW or self.conf["shirashi"] #i added this otherwise during volume aqisition no live image and image saving
```
Comment (vilim) [inline sashimi/state.py:468]: the conf should not be accessed at this point, the State can have boolean variables `camera_enabled`, `laser_enabled`, `scanning_enabled`, which can be used both here (you would check for `scanning_enabled`) and in the GUI construction (we would want to hide the widgets for components that are not enabled)

**Review by vilim — CHANGES_REQUESTED** [review]
Good start! I left some pointers how to improve it

File: sashimi/config.py (hunk 1 above)
Comment (kkoetter) [reply sashimi/config.py:30]: for reference: Chirashizushi (ちらし寿司, "scattered sushi"). I renamed it everywhere in my version. I plan to keep working on this for more functional for the LFM and would like the name to stay for now if thats okay.

File: sashimi/scanning.py (hunk 2 above)
Comment (kkoetter) [reply sashimi/scanning.py:497]: Will there be a disabling of the GUI for Mock/Disabled versions as well?

File: sashimi/state.py (hunk 3 above)
Comment (kkoetter) [reply sashimi/state.py:468]: will you work on this with regards to #79 as well or shall i do that?

**Pull request closed without merging**